packages/blitz/src/cli/commands/new.ts

```diff
@@ -126,7 +129,8 @@ const determineTemplate = async () => {
     (args["--template"] && !Object.keys(templates).includes(args["--template"].toLowerCase()))
   ) {
     const choices: Array<{value: keyof typeof templates; title: string}> = [
-      {value: "full", title: "Full - includes DB and auth (Recommended)"},
+      {value: "app", title: "App Router: Full Includes DB and auth (Recommended)"},
+      {value: "pages", title: "Pages Router: Full - includes DB and auth"},
       {value: "minimal", title: "Minimal — no DB, no auth"},
```

Comment:
```suggestion
      {value: "app", title: "App Router – Full Setup: includes DB and auth (Recommended)"},
      {value: "pages", title: "Pages Router – Full Setup: includes DB and auth"},
      {value: "minimal", title: "Pages Router – Minimal setup: no DB, no auth"},
```

I assume the minimal is `/pages`, yes? Then I suggest to update all three options